tools/distrib/python/grpcio_tools/grpc/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016, Google Inc.
+# Copyright 2015-2016, Google Inc.
```

Comment:
Am I reading this right that you're copying most of gRPC Python's library code into tools/distrib/python/grpcio_tools/grpc? Huh? What's up with that?